twilight-http/src/routing.rs

```diff
@@ -2623,11 +2623,15 @@ impl Display for Route<'_> {
                 f.write_str("?")?;
 
                 if *with_counts {
-                    f.write_str("with-counts=true")?;
+                    f.write_str("with_counts=true")?;
                 }
 
                 if *with_expiration {
-                    f.write_str("with-expiration=true")?;
+                    if *with_counts {
+                        f.write_str("&")?;
+                    }
+
+                    f.write_str("with_expiration=true")?;
```

Comment:
For the sake of consistency.
```suggestion
                    f.write_str("&with_expiration=true")?;
```
Webstandard allows queries like this too (`https://example.com/foo?&bar=true`), so there is no need to explicitly check whether we should add it or not.